pkg/kubefed2/join.go

```diff
@@ -534,6 +552,116 @@ func createClusterRoleBinding(clusterClientset client.Interface, saName, namespa
 	return &clusterRoleBinding, nil
 }
 
+// createRoleBinding creates an RBAC role and binding that allows the
+// service account identified by saName to access all resources in the
+// specified namespace.
+func createRoleBinding(clusterClientset client.Interface, saName, namespace,
```

Comment:
[Suggestion: Implementation not necessary in this PR to merge] Some portion of code in `createRoleBinding()` and `createClusterRoleBinding()` is common, which can be pushed to a common function and reused in both.

Reply:
I've posted a new PR that factors things out as you suggest: #222